### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by burnoutcomputer**
I spent way too much time fighting with the Nav Bar trying to center it before giving up.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```
Comment (mbMosman) [inline index.html]: This looks OK, but doesn't match what I see on the web: https://burnoutcomputer.github.io/portfolio/

You might check your email to make sure there wasn't an error or something publishing.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            </div>
+            <div class="col-xs-6 col-md-push-3 col-md-3">
+                <h3>Eric Nelson</h3>
+                <p>E-mail Address link
+                    <br> Phone number link</p>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-xs-6 col-md-push-2 col-md-3">
```
Comment (mbMosman) [inline index.html]: I'm not sure what you are trying to do with the push styles.  Is that to move it toward the center?  If so I think that makes sense... if not we should talk about this.

I'd also suggest that side by side on mobile is going to be too small unless you shrink text and have very, very short values.  I'd just suggest stacking them.

**Comment by mbMosman** [comment]
This is good for a start, but you need to have the links functional and use scrollspy to correctly implement the navigation with active links highlighted.

Also, you won't get any credit until you submit wireframes.  Get me acceptable wireframes and I'll record a non-zero grade for you.

File: index.html (hunk 2 above)
Comment (burnoutcomputer) [inline index.html]: It was to center the objects. I was fighting with the navbar and with
mobile styles, so I gave up and just pushed what I had working.

Comment (mbMosman) [inline index.html]: OK that's fine.  That actually works just fine, I wasn't sure though that was your intent.  Bootstrap intends for you to use offset for this: http://getbootstrap.com/css/#grid-offsetting

File: index.html (hunk 1 above)
Comment (burnoutcomputer) [inline index.html]: I did not receive an email about an error, and I am not sure what is going on there.

File: index.html (hunk 2 above)
Comment (burnoutcomputer) [inline index.html]: I was not aware of this. There is too much to bootstrap to learn and understand without more time and guidance on the subject.

**Pull request closed without merging**